CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
+
```

Comment:
remove this blank line, it is just needed one blank line at the end of the file, this has two